### DISCUSSION TITLE
Make key overlay customizable

### DISCUSSION THREAD

**Discussion by Uniter343** [ppy/osu#20625] [Ideas] [answered]
With the addition of smoke the key overlay now always shows 3 keys. For some people the 3rd key might never be used or they would simply prefer to not see it (it makes the overlay significantly larger). 

I know the overlay is not yet editable (movable) in the skin layout editor, so maybe this will happen once it is?

**Comment by peppy** [comment, marked as answer]
It will be redesigned as part of the upcoming HUD redesign.
